### ISSUE TITLE
"Guides" Dropdown in Header Navigation Not Accessible to Screen Readers

### ISSUE THREAD

**Issue by SlvrEagle23** [AzuraCast/azuracast.com#36]
See [reference tweet](https://twitter.com/crosswavesradio/status/1225875029235240961)

**Comment by SC2Mitch** [comment]
I've just checked this issue and it appears to be resolved with NVDA, while it's not 100% perfect, it does work albeit with a bit of delay hovering over the tabs.

**Comment by Vaalyn** [comment]
Since we have a new static website & the docs pages this issue is now invalid.
